### DISCUSSION TITLE
Permissions Style is not working

### DISCUSSION THREAD

**Discussion by robertbur** [requarks/wiki#3451] [Help / Questions]
Hello @everyone,

As I said in this post: https://github.com/Requarks/wiki/issues/2506, the access for the Style is not working. I set all the permissions to the users, and style not working.

Looking into more details looks like the permissions and at the page rules, the style is missing. 

**Host Info (please complete the following information):**
OS: [Docker]
Wiki.js version: [2.5.144]
Database engine: [MariaDb]

Apologies in advance for ask again. 

Kind Regards

**Comment by BasToTheMax** [comment]
Hi! #2506 is answered and closed

**Comment by robertbur** [comment]
Hi @BasToTheMax,

Affirm is closed, however, the answer that they sent to me doesn't solve my problem and I still looking for a solution for it.

Kind Regards
